Jenkinsfile

```diff
@@ -68,7 +68,7 @@ pipeline {
               ${ctestcmd}
             """
           }
-          parallel(linux_jobs + macos_jobs)
+          throttle(['long']) { parallel(linux_jobs + macos_jobs) }
```

Comment:
Could you please move the contents of the braces on its own line? So that we have one "building block" per line?

Reply:
sure